src/components/PickYourPositionSection/Carousel.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable react/jsx-props-no-spreading */
+/* eslint-disable jsx-a11y/alt-text */
+import React from 'react';
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+import Slider from 'react-slick';
+// import { CardDeck } from './CardDeck'
+import { PositionImage, PositionImageContainer } from './PositionsCSS';
+import DesktopcardsPivot from '../../images/DesktopcardsPivot.png'
+import DesktopcardsJammer from '../../images/DesktopcardsJammer.png'
+import DesktopcardsBlocker from '../../images/DesktopcardsBlocker.png'
+import DesktopcardsFreneso from '../../images/DesktopcardsFreneso.png'
+
+export const Carousel = () => {
```

Comment:
Maybe I missed the carousel-function on your page? But nice stretch goal to dive into!